python/lsst/ip/diffim/utils.py

```diff
@@ -1265,6 +1265,7 @@ def detectTestSources(exposure):
         sigma=None,  # The appropriate sigma is calculated from the PSF
         doSmooth=True
     )
+    exposure.mask.addMaskPlane("STREAK")  # Cannot run maskStreaksTask here, as it's in pipe_tasks
```

Comment:
Just curious why we need to know that we can't run maskStreakTask at this location. Is that something we might want to do?

Reply:
I see in another PR you do ` self.exposure.mask.addMaskPlane("STREAK")  # add empty streak mask plane in lieu of maskStreaksTask` as the explanation instead. Maybe swap to that?

Reply:
Good call for consistency's sake, will update. (I originally called the task here, and realized that wouldn't work since pipe_tasks explicitly depends on ip_diffim being setup, and the reverse cannot be true.)